### ISSUE TITLE
macOS disabled contextual menu Item's should be read by VoiceOver

### ISSUE THREAD

**Issue by Saadnajmi** [microsoft/fluentui-react-native#1208]
### Platform

- [ ] iOS
- [X] macOS
- [ ] win32
- [ ] windows
- [ ] android

### Describe the feature that you would like added

Made this so I have a bug number I can track. 
On system context menus, when VoiceOver is off, disabled menu items are not focusable. When VoiceOver is on, they are. We should mimic this in our own ContextualMenu component

### What component or utility would this be added to

ContextualMenu

### Additional context/screenshots

https://user-images.githubusercontent.com/6722175/144518159-ee88ffe1-ccf9-43af-aedc-00494a74cc63.mov
